src/main/webapp/app/entities/project/project.resolve.ts

```diff
@@ -33,6 +33,6 @@ export class ProjectResolve implements Resolve<IProject> {
       );
     }
 
-    return of(new Project());
+    return of({});
```

Comment:
Was bedeuten die geschweiften Klammern in der Methode? `of({})`

Reply:
Das ist ein leeres Objekt.
Frage mich aber gerade, ob es nicht ein `EMPTY` observable zurückgeben soll, dass somit nichts emitiert und sofort completed.

Reply:
Oder einfach ein Error werfen. Das wäre ja, wenn wir den Resolver an einem falschen Ort verbauen. Würde ich aber in einem separaten PR addressieren solche dinge Dinge.

Reply:
Ah okey. Danke für die Erklärung